[2025-05-27T01:11:28.929+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-27T01:11:28.952+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-27T01:11:24.526518+00:00 [queued]>
[2025-05-27T01:11:28.960+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-27T01:11:24.526518+00:00 [queued]>
[2025-05-27T01:11:28.960+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2025-05-27T01:11:28.970+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_api_data> on 2025-05-27 01:11:24.526518+00:00
[2025-05-27T01:11:28.977+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1531) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-27T01:11:28.979+0000] {standard_task_runner.py:63} INFO - Started process 1537 to run task
[2025-05-27T01:11:28.979+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_api_data', 'manual__2025-05-27T01:11:24.526518+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmp18f4ywrj']
[2025-05-27T01:11:28.980+0000] {standard_task_runner.py:91} INFO - Job 210: Subtask extract_api_data
[2025-05-27T01:11:29.065+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_api_data manual__2025-05-27T01:11:24.526518+00:00 [running]> on host 295abcd9c300
[2025-05-27T01:11:29.122+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_api_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T01:11:24.526518+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-27T01:11:24.526518+00:00'
[2025-05-27T01:11:29.123+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-27T01:11:29.127+0000] {etl.py:32} INFO - api data read
[2025-05-27T01:11:29.132+0000] {etl.py:35} INFO - api json data normalized
[2025-05-27T01:11:29.135+0000] {etl.py:37} INFO - api data source pickled
[2025-05-27T01:11:29.136+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-27T01:11:29.136+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-27T01:11:29.141+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_api_data, execution_date=20250527T011124, start_date=20250527T011128, end_date=20250527T011129
[2025-05-27T01:11:29.173+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-27T01:11:29.189+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-27T01:11:29.190+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
